src/components/tool/guide/Config.js

```diff
@@ -137,7 +60,7 @@ export const getCpuVendor = (config, onChange) => () => {
   const cpuVendors = ["Any", "AMD", "Intel"];
   return {
     element: (key, selected) => (
-      <ConfigGroup
+      <ConfigPicker
         key={key}
         title="CPU Vendor"
         type="picker"
```

Comment:
`type` prop no-longer needed

Reply:
Missed this. Fixed, also fixed wrong component